pkg/versions/versions.go

```diff
@@ -6,3 +6,10 @@ const (
 	DefaultSubmarinerVersion         = "0.7.0"
 	DefaultLighthouseVersion         = "0.7.0"
 )
+
+var (
```

Comment:
This complicates the code and we still would need to maintain those ^ constants.

My proposal is to simply use DefaultSubmarinerRepo, .... DefaultLighthouseVersion as variables and override those at compilation time.

Reply:
```var (
	DefaultSubmarinerRepo            = "quay.io/submariner"		DefaultSubmarinerRepo            = "quay.io/submariner"
	DefaultSubmarinerOperatorVersion = "0.7.0"		DefaultSubmarinerOperatorVersion = "0.7.0"
	DefaultSubmarinerVersion         = "0.7.0"		DefaultSubmarinerVersion         = "0.7.0"
	DefaultLighthouseVersion         = "0.7.0"		DefaultLighthouseVersion         = "0.7.0"
)	)

Reply:
Great! I believe we have to change the release process as well right?